strongbox-data-service/src/main/java/com/orientechnologies/orient/object/jpa/OJPAObjectDatabaseTxEntityManager.java

```diff
@@ -202,12 +199,42 @@ public Query createNativeQuery(String sqlString,
         throw new UnsupportedOperationException("createNativeQuery");
     }
 
+    @Override
+    public StoredProcedureQuery createNamedStoredProcedureQuery(String s)
+    {
+        return null;
```

Comment:
`return null;`

I believe we should `throw new UnsupportedOperationException` for each new method from the persistence API already not supported by us.

throw new UnsupportedOperationException("createNativeQuery");

Reply:
+1
also why this methods were added?